stories/useGravity2D.stories.tsx

```diff
@@ -48,7 +48,52 @@ export const Gravity2DBasic: React.FC = () => {
   );
 };
 
+export const Gravity2DCustomG: React.FC = () => {
+  const [center, setCenter] = React.useState<[number, number]>([0, 0]);
+
+  React.useLayoutEffect(() => {
+    const root = document.getElementById('root');
+    if (root) {
+      setCenter([root.clientWidth / 2, root.clientHeight / 2]);
+    }
+  }, []);
+
+  const [props] = useGravity2D({
+    config: {
+      attractorMass: number('attractorMass', 20),
+      moverMass: number('moverMass', 1),
```

Comment:
With a `G` of 0.4 our masses can now be a lot smaller to achieve a similar effect animation-wise.